pkg/usage/model_principal.go

```diff
@@ -0,0 +1,62 @@
+package usage
+
+import (
+	"time"
+
+	"github.com/Optum/dce/pkg/errors"
+	validation "github.com/go-ozzo/ozzo-validation"
+)
+
+// Principal item
+type Principal struct {
```

Comment:
Is this really a Principal? With fields like Date, CostAmount, Limit, etc., maybe this is a PrincipalUsage?